src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -70,30 +70,6 @@ void process(MirrorSynchronizedWebhookEvent event) {
                 .collect(Collectors.toSet());
     }
 
-    private boolean hasMatchingRepository(RefChangedDetails refChangedDetails, ParameterizedJobMixIn.ParameterizedJob job) {
```

Comment:
Are changes to this class intentional?

Reply:
Sorry about this. After I rebased to master, this was being flagged by the checkstyle so I opt-cmd-sft-L'd this to clean it up. Didn't realize how much that changed...

Reply:
Cool, so it's only formatting changes